dts: arm: st: Remove use of CONFIG_SOC_* from STM32 L0/L4 dts files

galak opened the pull request.

erwango commented on boards/arm/dragino_lsn50/dragino_lsn50.dts:
```diff
@@ -5,7 +5,7 @@
  */
```
You need to change your mail address in this commit

galak committed:
dts: arm: st: Remove use of CONFIG_SOC_* from STM32 L4 dts files

To move forward and remove use of Kconfig in dts files lets just create
SoC specific dtsi files that the boards can include.  We also seperate
out the L4 dtsi files into their own dir.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

galak committed:
dts: arm: st: Remove use of CONFIG_SOC_* from STM32 L0 dts files

To move forward and remove use of Kconfig in dts files lets just create
SoC specific dtsi files that the boards can include.  We also seperate
out the L0 dtsi files into their own dir.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>
